app/metrics.yaml

```diff
@@ -66,7 +66,7 @@ activation:
     data_sensitivity:
       - highly_sensitive
     notification_emails:
-      - jalmeida@mozilla.com
+      - android-probes@mozilla.com
     expires: never
```

Comment:
@jonalmeida for the purpose of permanent collections, data-review requires an individual contributor's email address to be responsible for the collection.

Reply:
Thanks for clarifying! I've made the change for both comments.